lib/install/management/network.go

```diff
@@ -95,26 +99,47 @@ func (d *Dispatcher) removeNetwork(conf *config.VirtualContainerHostConfigSpec)
 		return nil
 	}
 
-	name := conf.Name
-	if network, err := d.session.Finder.Network(d.ctx, name); err != nil || network == nil {
-		log.Infof("Didn't find network %q", name)
-		log.Debugf("Didn't find network for %s", err)
+	log.Debugf("Remove bridge network based on %s", conf.ExecutorConfig.Networks["bridge"].Network.ID)
+
+	name := conf.ExecutorConfig.Networks["bridge"].Network.ID
+	moref := new(types.ManagedObjectReference)
```

Comment:
Why did you create a pointer to an object instead of just object? Looking at usage you just dereference it creating a copy.